eslint.config.mjs

```diff
@@ -0,0 +1,78 @@
+import js from '@eslint/js'
+import eslintPluginGithub from 'eslint-plugin-github'
+import stylistic from '@stylistic/eslint-plugin'
+
+export default [
+  eslintPluginGithub.getFlatConfigs().recommended,
+  js.configs.recommended,
+  ...eslintPluginGithub.getFlatConfigs().typescript,
+  {
+    settings: {
+      'import/resolver': {
+        node: {
+          extensions: ['.ts'],
+        },
+      },
+    },
+  },
+  {
+    languageOptions: {
+      parserOptions: {
+        ecmaVersion: 9,
+        sourceType: 'module',
+        project: './tsconfig.json',
+      },
+    },
+    plugins: {
+    '@stylistic': stylistic,
+    }
+  },
+  {
+    rules: {
+      'sort-imports': 'off',
+      'i18n-text/no-en': 'off',
+      'eslint-comments/no-use': 'off',
+      'import/no-namespace': 'off',
+      'no-unused-vars': 'off',
+      '@typescript-eslint/no-unused-vars': 'error',
+      '@typescript-eslint/explicit-member-accessibility': [
+        'error',
+        { accessibility: 'no-public' },
+      ],
+      '@typescript-eslint/no-require-imports': 'error',
+      '@typescript-eslint/array-type': 'error',
+      '@typescript-eslint/await-thenable': 'error',
+      '@typescript-eslint/ban-ts-comment': 'error',
+      camelcase: 'off',
+      '@typescript-eslint/consistent-type-assertions': 'error',
+      '@typescript-eslint/explicit-function-return-type': [
+        'error',
+        { allowExpressions: true },
+      ],
+      'func-call-spacing': ['error', 'never'],
+      '@typescript-eslint/no-array-constructor': 'error',
+      '@typescript-eslint/no-empty-interface': 'error',
+      '@typescript-eslint/no-explicit-any': 'error',
+      '@typescript-eslint/no-extraneous-class': 'error',
+      '@typescript-eslint/no-for-in-array': 'error',
+      '@typescript-eslint/no-inferrable-types': 'error',
+      '@typescript-eslint/no-misused-new': 'error',
+      '@typescript-eslint/no-namespace': 'error',
+      '@typescript-eslint/no-non-null-assertion': 'warn',
+      '@typescript-eslint/no-unnecessary-qualifier': 'error',
+      '@typescript-eslint/no-unnecessary-type-assertion': 'error',
+      '@typescript-eslint/no-useless-constructor': 'error',
+      '@typescript-eslint/no-var-requires': 'error',
+      '@typescript-eslint/prefer-for-of': 'warn',
+      '@typescript-eslint/prefer-function-type': 'warn',
+      '@typescript-eslint/prefer-includes': 'error',
+      '@typescript-eslint/prefer-string-starts-ends-with': 'error',
+      '@typescript-eslint/promise-function-async': 'error',
+      '@typescript-eslint/require-array-sort-compare': 'error',
+      '@typescript-eslint/restrict-plus-operands': 'error',
+      semi: ["error", "never"],
+      '@stylistic/type-annotation-spacing': 'error',
+      '@typescript-eslint/unbound-method': 'error',
```

Comment:
```suggestion
      "semi": "off",
      "@typescript-eslint/semi": ["error", "never"],
      "@typescript-eslint/type-annotation-spacing": "error",
      "@typescript-eslint/unbound-method": "error"
```

I think there was a copy/paste error here ^

Reply:
I added only `semi": ["error", "never"],` because it looks like `semi` cannot be found in the `@typescript-eslint` plugin. So it's either ` semi: 'off',` or `semi": ["error", "never"],`.

Reply:
Ah got it, I see: https://typescript-eslint.io/rules/semi/ - looks like it got moved. 

Okay, sounds good.